src/advertising_categories.h

```diff
@@ -0,0 +1,211 @@
+#include <map> 
+
+std::map<std::string, int> get_advertising_categories(){
+    std::map<std::string,int> advertising_categories = {};
```

Comment:
nit:
```
std::map<std::string, int> advertising_categories = {
  {"architecture-architecture", 0},
  {"arts & entertainment-animation", 1},
  {"arts & entertainment-anime", 2},
  ...
};
```

Reply:
I would like to nuke this file entirely and just return key-values when this is merged, also referring to https://github.com/brave-intl/bat-native-usermodel/pull/25#discussion_r390191221